static/js/tools/statsvar_menu.tsx

```diff
@@ -154,7 +154,7 @@ class Node extends Component<NodePropType, NodeStateType> {
   private _handleHashChange() {
     const statsVarPathNext = [];
     let check = false;
-    let expand = false;
+    let expand = this.state.expanded;
```

Comment:
To confirm, once a node is expanded, it will keep expanded

Reply:
Right. Unless the user manually close it, or the page is refreshed.